templates/inspec/doc-template.md.erb

```diff
@@ -0,0 +1,19 @@
+<%= lines(autogen_notice :markdown) -%>
+
+---
+title: About the <%= object.name -%> resource
+platform: gcp
+---
+
+## Syntax
+A `<%= resource_name(object, product_ns) -%>` is used to test a Google <%= object.name -%> resource
+
+TODO: Examples
+
+## Properties
+Properties that can be accessed from the `<%= resource_name(object, product_ns) -%>` resource:
+<% object.properties.each do |prop| -%>
+  * `<%= "#{prop.out_name}" -%>`: <%= "#{prop.description}" -%>
+
+<%= sub_property_descriptions(prop) -%>
```

Comment:
What if the nested property isn't the last property in this list? Wouldn't it create inconsistent newlines?

(It looks like every non-nested object gets a newline, but nested-objects don't)

Reply:
Looking at a doc that has been generated that contains more nested objects than zone it actually works surprisingly well. Only issue I notice is that newlines within a property description cause issues. Will definitely revisit this once we start generating more complex resources, but I'm inclined to leave it in this state for now

Reply:
Can you take a look at it now? Even now in the Markdown files, there's inconsistent spacing

Reply:
Should format consistently for nested properties & top level properties. Probably need to make it recursive at some point to support deeper nesting